### ISSUE TITLE
NMS-related error when using ParticleArgument class

### ISSUE THREAD

**Issue by CubBossa** [JorelAli/CommandAPI#383]
### CommandAPI version

8.7.0

### Minecraft version

1.19.3

### Are you shading the CommandAPI?

Yes

### What I did

- I initialized the API
```Java
		CommandAPI.onLoad(new CommandAPIConfig()
				.verboseOutput(configuration.isVerbose())
				.useLatestNMSVersion(true)
				.initializeNBTAPI(NBTContainer.class, NBTContainer::new));
```
- I used a ParticleArtgument in a CommandTree
 
```Java
return super.appendEditCommand(tree, visualizerIndex, argumentOffset)
				.then(new LiteralArgument("particle")
						.then(new ParticleArgument("particle")
								.executes((commandSender, objects) -> {
									ParticleVisualizer visualizer = (ParticleVisualizer) objects[visualizerIndex];
									onSetParticle(commandSender, visualizer, (ParticleData) objects[argumentOffset], null, null, null);
								})
```

- I built the plugin as usually with maven and started the server

### What actually happened

An error occured when loading the plugin. This did not happen before while the code did not change.

```Log
[21:42:26 INFO]: [CommandAPI] Chat preview is not available
[21:42:26 ERROR]: Error occurred while enabling PathFinder v1.5.2 (Is it up to date?)                                                                                                                          java.lang.NoSuchMethodError: 'net.minecraft.commands.arguments.ArgumentParticle net.minecraft.commands.arguments.ArgumentParticle.a()'                                                                                 at de.cubbossa.pathfinder.commandapi.nms.NMS_Common._ArgumentParticle(NMS_Common.java:254) ~[?:?]                
at de.cubbossa.pathfinder.commandapi.arguments.ParticleArgument.<init>(ParticleArgument.java:41) ~[?:?]          
at de.cubbossa.pathfinder.module.visualizing.visualizer.ParticleVisualizerType.appendEditCommand(ParticleVisualizerType.java:49) ~[?:?]
at de.cubbossa.pathfinder.module.visualizing.command.PathVisualizerCommand.register(PathVisualizerCommand.java:76) ~[?:?] 
at de.cubbossa.pathfinder.PathPlugin.onEnable(PathPlugin.java:243) ~[?:?]                                        
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]     
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:371) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:544) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugin(CraftServer.java:578) ~[paper-1.19.3.jar:git-Paper-345]       
at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugins(CraftServer.java:492) ~[paper-1.19.3.jar:git-Paper-345]      
at org.bukkit.craftbukkit.v1_19_R2.CraftServer.reload(CraftServer.java:1038) ~[paper-1.19.3.jar:git-Paper-345]   
at org.bukkit.Bukkit.reload(Bukkit.java:930) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]                             
at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:54) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]       
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]        
at org.bukkit.craftbukkit.v1_19_R2.CraftServer.dispatchCommand(CraftServer.java:929) ~[paper-1.19.3.jar:git-Paper-345]    
at org.bukkit.craftbukkit.v1_19_R2.CraftServer.dispatchServerCommand(CraftServer.java:892) ~[paper-1.19.3.jar:git-Paper-345]
at net.minecraft.server.dedicated.DedicatedServer.bg(DedicatedServer.java:494) ~[paper-1.19.3.jar:git-Paper-345] 
at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:441) ~[paper-1.19.3.jar:git-Paper-345]  
at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1397) ~[paper-1.19.3.jar:git-Paper-345]           
at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1173) ~[paper-1.19.3.jar:git-Paper-345]           
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-345]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### What should have happened

Plugin load as usual

**Comment by JorelAli** [comment]
NMS implementation changed in 1.19.3 and this went unnoticed. Fixed in 937c7966e8a8af2a19ae675287c386f733322a29 for 8.7.1.

**Comment by JorelAli** [comment]
Fixed in 8.7.1.
